Rakefile

```diff
@@ -18,7 +18,7 @@ require_relative 'tasks/assets'
 require_relative 'tasks/profiles'
 require_relative 'tasks/geo'
 
-Mongoid.load!('./config/mongoid.yml')
+Mongoid.load!('./config/mongoid.yml', :test)
```

Comment:
`RACK_ENV` should be just fine. See `Mongoid attempted to find the appropriate environment but no Rails.env, Sinatra::Base.environment, RACK_ENV, or MONGOID_ENV could be found.`
I don't know what's wrong here...